### PR TITLE
send all transition context params, not just the first one

### PR DESCRIPTION
**Pull request by kielni**
leaving out params causes "Uncaught Error: You must provide param model_id to generate."
when generating URL from measureTransition

fix for https://github.com/mike-north/ember-perf/issues/101

### CODE REVIEW

**Comment by kiwiupover** [comment]
Thanks @kielni 

Can you add a test for this fix?

Cheers
Dave

File: tests/acceptance/event-body-test.js [hunk 1]
```diff
@@ -42,6 +42,29 @@ test('Initial load, then drilling in', function(assert) {
 
 });
 
+test('More than one route context object', function(assert) {
```
Comment (kiwiupover) [inline tests/acceptance/event-body-test.js]: Wonderful thanks heaps!

**Comment by kiwiupover** [comment]
@kielni some how the test didn't run on CI you mind either pushing a noop commit or squash the commits and push again. Sorry about this.

**Comment by kiwiupover** [comment]
@kielni the tests failed on ember 1.11 and 1.12 are you able to have a look at that?

@mike-north @stefanpenner should we drop support for pre `1.13`?

**Comment by kielni** [comment]
I saw that, but I can't tell what's wrong: `The value that #each loops over must be an Array. You passed [1,2,3,4]`  

I think that's from https://github.com/kielni/ember-perf/blob/51e08a18f52f5bd14217a9a7aaaa74166bd8b58d/tests/dummy/app/templates/companies.hbs#L5

**Comment by kiwiupover** [comment]
@kielni let's wait and see what @mike-north and @stefanpenner think about dropping pre ember `1.13` versions.

**Comment by kielni** [comment]
It looks like it's just a problem with the dummy app test fixture in the older Ember versions.  I could just put a direct link to building 1 instead of looping over the buildings; what do you think?

**Comment by kiwiupover** [comment]
Thanks heaps @kielni

**Pull request merged**